system/lib/libc/musl/src/thread/pthread_cond_timedwait.c

```diff
@@ -187,10 +187,6 @@ int __pthread_cond_timedwait(pthread_cond_t *restrict c, pthread_mutex_t *restri
 		__pthread_setcancelstate(PTHREAD_CANCEL_DISABLE, 0);
 	}
 
-#ifdef __EMSCRIPTEN__
-	pthread_testcancel();
-#endif
-
```

Comment:
I removed this since it looked redundant. This was introduced in commit https://github.com/emscripten-core/emscripten/commit/d2bb705cea3a434a936656cd51da889a08be07eb (which was based on musl 1.0.5), but when musl was updated to 1.1.15 (commit https://github.com/emscripten-core/emscripten/commit/31cf2bb1bc3a501a14ea9647f2bc7ad25fb2fce4), it ends up calling `__pthread_testcancel` twice, see:
https://github.com/emscripten-core/emscripten/blob/7e6d511d91ff593c3ec741857fb6d33dbcce59e6/system/lib/libc/musl/src/thread/pthread_cond_timedwait.c#L185-L192

If preferred, I can also create a separate PR for this.